app/src/main/java/an/maguste/android/navier/FragmentMoviesList.kt

```diff
@@ -49,7 +63,14 @@ class FragmentMoviesList : Fragment() {
         recycler?.hasFixedSize()
 
         // "upload" movies data
-        setMovieData()
+        getMovieData()
+    }
+
+    private fun getMovieData() {
+        scope.launch {
+            moviesList = context?.let { loadMovies(it) }
```

Comment:
Почти тоже самое. Проверки контекста на null реализованы внутри `requireContext()`, в итоге - твой код без ветвлений.
```suggestion
            moviesList = loadMovies(requireContext())
```